dev-days-2019/README.md

```diff
@@ -24,20 +24,32 @@ Now that you have SUSHI installed and a folder containing FSH files, you can run
 ```
 $ sushi <path>
 ```
-where `<path>` is the path to the folder containing the FSH files. So if your FSH files were in `~/fhir-shorthand/dev-days-2019/working-samples`, the command would be
+where `<path>` is the path to the folder containing the FSH files. So if change the working directory to `/dev-days-2019`, the command would be
 ```
-$ sushi ~/fhir-shorthand/dev-days-2019/working-samples
+$ sushi .
 ```
-This will send the resulting FHIR output to an `out` directory in your current working directory. Optionally, you can specify your output directory name using the `-o` option.
+This will send the resulting FHIR output to an `build` directory in your current working directory. Optionally, you can specify your output directory name using the `-o` option.
 ```
 $ sushi <path> -o <output-directory>
 ```
 When running SUSHI successfully, you should see output similar to the following
 ```
 info: Exported 4 profile(s) and 1 extension(s).
 ```
-### Step 5: Experiment
+### Step 5: Generate an IG
+
+Check to see if the /build directory (or the directory you specified) is present. 
+
+Now change directory of the command window to the output directory, `/dev-days-2019/build`. At the command prompt, enter:
+
+```
+$ _genonce
+```
```

Comment:
It complicates things, but since you have a `$` prompt in the above example, I think the command technically needs to be:
```sh
$ ./_genonce.sh
```

Also, you're missing the previous step, which is that the first time, you need to run:
```sh
$ ./updatePublisher.sh
```

Reply:
ok, fixed.